.craft.yml

```diff
@@ -30,12 +30,14 @@ targets:
     bucket: sentry-sdk-assets
     paths:
       - path: /sentry-cli/{{version}}/
-        maxCacheAge: 2592000
+        metadata:
+          cacheControl: 'public, max-age=2592000'
       - path: /sentry-cli/latest/
-        maxCacheAge: 600
+        metadata:
+          cacheControl: 'public, max-age=600'
   - name: registry
     type: app
     urlTemplate: "https://downloads.sentry-cdn.com/sentry-cli/{{version}}/{{file}}"
-    includeNames: /^.*sentry-cli.*$/
+    includeNames: /^sentry-cli-.*$/
```

Comment:
Seeing this change, this would be a good point to consider tagging artifacts or relying on the MIME type, maybe? Should not block this PR, but I highly encourage to consider this. Otherwise, we might get false-positives if we coincidentally add another artifact starting with `sentry-cli-`, which is not so unlikely.

Reply:
You're totally right, this should be definitely implemented at some point.